app.py

```diff
@@ -80,7 +80,8 @@ def vote(vote_button, response_a, response_b, model_a_name, model_b_name,
 """
 
 with gr.Blocks(title="Yanolja Arena", css=css) as app:
-  set_token(app)
+  token = gr.Textbox(visible=False)
```

Comment:
I have the same questions as others.
1. If it's saved here in this variable which live in memory, and this token is used through the whole app, what's the reason why you save it in local storage?
2. What's the reason why you save it in local storage when you don't re-use it? like, the initialization of token should be.. `get token from local storage -> set it as a token -> use it..`? isn't it?

Reply:
you're right. I'll update the code thank you!